dist/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <title>Restaurant Page</title>
+</head>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+    <a class="navbar-brand" href="#">Restaurant</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+  
+    <div class="collapse navbar-collapse" id="navbarColor01">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
+          <a class="nav-link" id="homePage" href="#">Home
+            <span class="sr-only">(current)</span>
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" id="menuPage" href="#">Menu</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" id="contactPage" href="#">Contact</a>
+        </li>
+      </ul>
```

Comment:
- [ ] As stated previously and also required in the below screenshot. Please replace all these elements with `<div id="content"></div>`. Create all the other HTML elements with the `document.createElement` and `document.appendChild`.
![ice_screenshot_20201123-152439](https://user-images.githubusercontent.com/33323077/99974072-bd5df200-2da0-11eb-9435-c3b83058b123.png)

Reply:
Hey ^^
That was more clear, thank you.
I've done the requested change into the code. Navbar and nav/tab buttons are created dynamically by JS any more. There is nothing else than "content div" on HTML.

Reply:
Glad that helped you. 💪